e2e/playwright-visual.config.js

```diff
@@ -4,13 +4,13 @@
 
 /** @type {import('@playwright/test').PlaywrightTestConfig<{ theme: string }>} */
 const config = {
-    retries: 1, // visual tests should never retry due to snapshot comparison errors. Leaving as a shim
+    retries: 1, // Visual tests should never retry due to snapshot comparison errors. Leaving as a shim
```

Comment:
I think we can set this back to `0` now since the Percy timeout bug is resolved.